### ISSUE TITLE
`unused variable` error while accessing imports in `export` statement

### ISSUE THREAD

**Issue by zdychacek** [palantir/tslint#818]
Hi,

I am getting following error when linting this code:

Code:
```ts
import A from './a;
import B from './b';

export { A, B };
```

Errors:
```
[1, 8]: unused variable: 'A'
[2, 8]: unused variable: 'B'
```

Using `tslint v3.0.0`.

O.

**Comment by zdychacek** [comment]
This problem does not apply exclusively to `export` statement.

Code:

```
import a from './a';
const obj = { a };
```

Error:

```
[1, 8]: unused variable: 'a'
```

O.

**Comment by JKillian** [comment]
Hi @zdychacek, thanks for the report. This is a duplicate of #684, so I'm going to close this issue. This is something that should be solved though!
